src/Commands/Debug/DebugMode.php

```diff
@@ -43,7 +43,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $isDebugModEnabled = $debugMode->isDebugModeEnabled();
 
         //check if action is allowed
-        if ( !in_array($action,$this->allowed_command_states)){
+        if ( !in_array($action,self::ALLOWED_COMMAND_STATES)){
```

Comment:
good practice is to add a space after a comma in order to follow the psr-2 function/method declaration standard